### PR TITLE
Removed partial load hints

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php [hunk 1]
```diff
@@ -532,17 +532,16 @@ protected function getPreview($id, $locale)
...
                         }
-                        break;
                     }
                 }
-                break;
             }
         }
 
-        return $media;
+        return null;
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php]: Why did this change?

Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php]: `$media` is a "running" var and should not be returned ... so i have changed the return statement in the loop and return null here!

**Comment by danrot** [comment]
A contact requires about 20 queries to be loaded, because the `ContactManager` uses the default `find` method of the repository: https://github.com/sulu-io/sulu/blob/28006c5212d391fdaec3ba98f8e60447e9988bc4/src/Sulu/Bundle/ContactBundle/Contact/ContactManager.php#L381

I am merging it anyway, since it is not introduced in this PR, but we have to update this behavior...

**Comment by wachterjohannes** [comment]
@danrot do you have created an issue for that?

**Pull request merged**